java/src/jmri/jmrit/operations/router/Router.java

```diff
@@ -76,7 +76,7 @@ public String getStatus() {
      * 
      * @param car         the car being tested
      * @param train       the first train servicing the car, can be null
-     * @param track       the destination track
+     * @param track       the destination track, can not be null
```

Comment:
If `track` must not be null, it's recommended to annotate it with `@Nonnull`.

`import javax.annotation.Nonnull;`

Example:
`public default AddressedProgrammer getAddressedProgrammer(@Nonnull DccLocoAddress address) {`